### DISCUSSION TITLE
QRAVE 1.0 - Launch Timing and Discussion

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/QRAVEPlugin#77] [Specification Discussion] [answered]
@MattReimer and @philipbaileynar, this is related to a broader series of updates and release schedule and testing around the Warehouse 3.x refactor. I want to discuss:

- The release of QRAVE 1.X as "Riverscape-Compliant", Professional Grade tool with report card, to coincide with launch of Warehouse 3.x.
- The release schedule of QRAVE 0.X updates that reflect the minor fixes, outstanding bug fixes (we need to prioritize) and internal testing (within lab) to iterate on getting these things dialed.
- The associated curation tasks we need to specify for the lab in [RiverscapesXML](https://github.com/Riverscapes/RiverscapesXML). 
- Setting up the milestones and release schedule around this and rallying troops.

**Comment by MattReimer** [comment, marked as answer]
One of the easy roadblocks here is that we shouldn't do this before soft-launching the new warehouse and migrating all the projects. 

- Any blocking or irritating bugs in QRave itself need to be prioritized and solved. 
- We were trying to make the `project.rs.xml` file upgrade process seamless but there are some potentially breaking changes we can't work around so keeping this `experimental` gives me the freedom to ignore backward compatibility (a little anyway).
- Any outstanding challenges we know about to RiverscapesXML (internationalization, tool versioning etc.) need to be discussed. Any deal breakers we can solve need to be tackled first.

I guess I'm not totally sure if we're launching QRave on its own or if it's a surrogate for launching RiverscapesXML and all connected tools in a broader sense.
